kolibri/utils/options.py

```diff
@@ -498,6 +507,15 @@ def url_prefix(value):
                 data that is returned to our telemetry server.
             """,
         },
+        "MINIMUM_DISK_SPACE": {
+            "type": "bytes",
+            "default": "250MB",
+            "description": """
+                The minimum free disk space that Kolibri should try to maintain on the device. This will
+                be used as the floor value to prevent Kolibri completely filling the disk during file import.
+                Value can either be an integer in bytes or suffixed with a unit from B, KB, MB, GB, TB, or PB.
```

Comment:
It appears that the suffixed versions can be fractional.

```suggestion
                Value can either be a number suffixed with a unit (e.g. MB, GB, TB) or an integer number of bytes.
```

While technically we may allow KB or PB, not sure it's helpful to mention here.